Mail/Views/Settings/SettingsOptionView.swift

```diff
@@ -34,6 +34,9 @@ struct SettingsOptionView<OptionEnum>: View where OptionEnum: CaseIterable, Opti
     private let matomoValue: Float?
     private let matomoName: KeyPath<OptionEnum, String>?
 
+    @LazyInjectService private var rootViewManager: RootViewManageable
```

Comment:
Not needed

Reply:
Since a refactor, the entire RootViewManageable is no longer needed. I removed all of it.